MediaBrowser.Model/Configuration/EncodingOptions.cs

```diff
@@ -136,6 +137,11 @@ public EncodingOptions()
     /// </summary>
     public string VaapiDevice { get; set; }
 
+    /// <summary>
+    /// Gets or sets the QSV device.
+    /// </summary>
+    public string QsvDevice { get; set; }
```

Comment:
This will confuse QSV users on Windows. Please add support for it as well.

Users who modify this in settings should be aware of what they are doing, as this skips the vendor id check.

Reply:
This is not passed on Windows at all so there will be no issues, just document it that on Windows this option will be ignored.

Reply:
> This is not passed on Windows at all so there will be no issues

So this only solves half of the problem - unable to select the index of QSV sub-device on both Linux & Windows.

Reply:
This has taken me awhile to get to, I don't actively have a windows device to be able to test this.
I'm having trouble targeting the correct device in windows using d3d11va
From my experience, it seems to split the load between the CPU and GPU.
When I try and set deviceIndex=1 I get an ffmpeg error and the stream crashes.
Testing with deviceIndex=0 yields the same results as the current implementation which just uses the vendor id.

I'm not sure how to modify the windows path in order to only target my intel GPU

Reply:
For Windows, just allow `QsvDevice` to be a number >= 0.
If you don't have dual Intel graphics cards on a system, setting this to 1 will crash qsv.
e.g.
0 -> UHD770 iGPU
1 -> Arc A770 dGPU